pkg/modeling/modeling.go

```diff
@@ -16,12 +16,14 @@ import (
 
 var (
 	mu                  sync.Mutex
-	modelSortings       [][]resource.Quantity
 	defaultModelSorting []clusterapis.ResourceName
 )
 
 // ResourceSummary records the list of resourceModels
-type ResourceSummary []resourceModels
+type ResourceSummary struct {
+	Rms           []resourceModels
```

Comment:
I think it is better to rename as `RMs` to represent `ResourceModels`, for `Rms` confused me at a galance.

Reply:
Done